CONTRIBUTING.md

```diff
@@ -7,6 +7,8 @@ Exercism would be impossible without people like you being willing to spend time
 
 You'll need Go version 1.7 or higher. Follow the directions on http://golang.org/doc/install
 
+You will also need `dep`, the Go dependency management tool. Follow the directions on https://github.com/golang/dep
```

Comment:
Let's link directly to the install docs `https://golang.github.io/dep/docs/installation.html`